src/QMCHamiltonians/ACForce.cpp

```diff
@@ -47,21 +48,27 @@ std::unique_ptr<OperatorBase> ACForce::makeClone(ParticleSet& qp, TrialWaveFunct
 std::unique_ptr<OperatorBase> ACForce::makeClone(ParticleSet& qp, TrialWaveFunction& psi_in, QMCHamiltonian& ham_in)
 {
   std::unique_ptr<ACForce> myclone = std::make_unique<ACForce>(ions_, qp, psi_in, ham_in);
+  myclone->fastDerivatives_=fastDerivatives_;
+  myclone->useSpaceWarp_=useSpaceWarp_;
+  myclone->first_force_index_=first_force_index_;
   return myclone;
 }
 
 bool ACForce::put(xmlNodePtr cur)
 {
   std::string useSpaceWarpString("no");
   std::string ionionforce("yes");
+  std::string fasteval("no");
   RealType swpow(4);
   OhmmsAttributeSet attr;
   attr.add(useSpaceWarpString, "spacewarp"); //"yes" or "no"
   attr.add(swpow, "swpow");                  //Real number"
   attr.add(delta_, "delta");                 //Real number"
+  attr.add(fasteval,"fast_derivatives");
   attr.put(cur);
 
   useSpaceWarp_ = (useSpaceWarpString == "yes") || (useSpaceWarpString == "true");
+  fastDerivatives_ = (fasteval == "yes") || (fasteval == "true");
```

Comment:
boolean can be `add()` directly without going through a string. Please update the above two.

Reply:
Sorry. I'm being dense.  Could you clarify or point to a code example?

Reply:
See do_param_output change in https://github.com/QMCPACK/qmcpack/pull/4221/files

Reply:
Oh that's slick.  Made those changes.  Done.